vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2334,6 +2338,16 @@ def get_kv_cache_spec(self) -> dict[str, KVCacheSpec]:
                         dtype=self.kv_cache_dtype,
                         sliding_window=attn_module.sliding_window,
                         use_mla=use_mla)
+                elif self.attention_chunk_size is not None \
```

Comment:
Can you implement or add assertion for both sliding window and chunked attention are used?